Externals.cfg

```diff
@@ -15,9 +17,10 @@ local_path = components/cice
 required = True
 
 [cime]
-tag = cime5.6.37
+#tag = cime5.6.37
+branch = maint-5.6-smyleL83
 protocol = git
-repo_url = https://github.com/ESMCI/cime
+repo_url = https://github.com/jedwards4b/cime
```

Comment:
Can we get these changes in your branch merged to maint-5.6.  I'm not a fan of using developers branches for release candidate tags.

Reply:
It's best to let me handle the Externals file.  I usually only update it when I start making a new cesm tag.